modules/sdk-coin-bsc/src/lib/keyPair.ts

```diff
@@ -0,0 +1,66 @@
+import { randomBytes } from 'crypto';
```

Comment:
Most of the code in this file and in the tests are the same as the coressponding files in `sdk-coin-avaxc`.